weblate/api/serializers.py

```diff
@@ -90,15 +90,35 @@ def __init__(self, *args, **kwargs):
 
 class LanguageSerializer(serializers.ModelSerializer):
     web_url = AbsoluteURLField(source="get_absolute_url", read_only=True)
+    plural = serializers.SerializerMethodField()
+    aliases = serializers.SerializerMethodField()
+    statistics_url = serializers.HyperlinkedIdentityField(
+        view_name="api:language-statistics", lookup_field="code"
+    )
 
     class Meta:
         model = Language
-        fields = ("code", "name", "direction", "web_url", "url")
+        fields = (
+            "code",
+            "name",
+            "plural",
+            "aliases",
+            "direction",
+            "web_url",
+            "url",
+            "statistics_url",
+        )
         extra_kwargs = {
             "url": {"view_name": "api:language-detail", "lookup_field": "code"},
             "code": {"validators": []},
         }
 
+    def get_plural(self, obj):
```

Comment:
```suggestion
    def get_plurals(self, obj):
```

It returns all plural rules, not just one ;-)